hack/copy-kube-artifacts.sh

```diff
@@ -24,20 +26,26 @@ rsync -av \
   --include-from=- \
   --exclude='*' \
   $KUBE_ROOT/ $KUBE_GODEP_ROOT <<EOF
+test/e2e/**/*.yaml
+test/e2e/**/*.json
 api/swagger-spec/v1.json
 cmd/integration/**.json
 cmd/integration/**.yaml
 docs/admin/**.json
 docs/admin/**.yaml
-docs/user-guide/**.json
-docs/user-guide/**.yaml
+docs/user-guide/**/*.json
```

Comment:
Nope, it's taking everything starting from `docs/user-guide/` and going down. Try for yourself with:
```
shopt -s globstar
ls Godeps/_workspace/src/k8s.io/kubernetes/docs/user-guide/**/*.yaml
```